### PR TITLE
only chmod files if they exist so we prevent the jenkins pod crash looping

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

File: postStart.sh [hunk 1]
```diff
@@ -46,5 +46,10 @@ if [[ -d "/root/repositoryscripts/src" && -d "/root/repositoryscripts/vars" ]];
 
 fi
 
-chmod -R 700 /root/.ssh
-chmod -R 700 /home/jenkins/.ssh-git
+if [ -f "/root/.ssh" ]; then
```
Comment (jimmidyson) [inline postStart.sh]: Shouldn't this be `-d` (directory) rather than `-f` (regular file, not directory?)

File: postStart.sh [hunk 2]
```diff
@@ -46,5 +46,10 @@ if [[ -d "/root/repositoryscripts/src" && -d "/root/repositoryscripts/vars" ]];
...
+  chmod -R 700 /root/.ssh
+fi
+
+if [ -f "/home/jenkins/.ssh-git" ]; then
```
Comment (jimmidyson) [inline postStart.sh]: As above.

File: postStart.sh (hunk 1 above)
Comment (rawlingsj) [inline postStart.sh]: yeah - I tested it without the keys there so it worked - will update

Comment (rawlingsj) [inline postStart.sh]: done https://github.com/fabric8io/jenkins-docker/pull/79

**Pull request merged**